mitxpro/templates/terms_of_services.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+{% load render_bundle %}
+{% load make_fragment_id %}
+{% load wagtailcore_tags %}
+
+{% block headercontent %}
+<div id="header"></div>
+{% render_bundle 'header' %}
+{% endblock %}
+
+{% block content %}
+
+<div class="container tos-container">
+  <div class="row tos-banner">
+    <h1 class="text-uppercase">Privacy Statement</h1>
+    <h2>A supporting statement of your proposition</h2>
```

Comment:
Shouldn't this be configurable from the CMS?

Reply:
@pdpinch do we want this change ?